### ISSUE TITLE
<kbd> tag gets rewritten as <code>

### ISSUE THREAD

**Issue by JVApen** [doxygen/doxygen#11209]
**Describe the bug**
The `<kbd>` tag is intended to indicate a keyboard key, however, when used in doxygen, it gets converted into `<code>` instead.

**Screenshots**
![image](https://github.com/user-attachments/assets/603b9b1a-c791-4603-8335-2481d74dacc7)
![image](https://github.com/user-attachments/assets/bab0f0ca-c1fb-4d46-a4c7-9191b4c56fd5)

**To Reproduce**
````{.cpp}
/*!
<kbd>Ctlr</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
*/
````

**Expected behavior**
I expect this to be rendered similarly to what you can see on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd
aka: <kbd>Ctlr</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>

**Version**
1.9.8 (c2fe5c3e4986974eb2a97608b24086683502f07f)

**Additional context**
https://www.doxygen.nl/manual/htmlcmds.html Unfortunately describes this tag as `Starts and ends a piece of text displayed in a typewriter font.`
Strangely enough, the standard doxygen css does contain formatting for `<kbd>`

**Comment by albert-github** [comment]
Currently doxygen uses for `<kbd>`  and `<code>` the same rendering also because other output formats have less flexible possibilities.

I created a small HTML file:
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<head>
</head>
<body>
<p>Test kbd tag</p>
<p>
With kbd tags:<br>
<kbd>Ctlr</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
<br>
With code tags:<br>
<code>Ctlr</code>+<code>Shift</code>+<code>P</code>
</p>
</body>
</html>
```
which renders as:
![image](https://github.com/user-attachments/assets/8ebe7816-3992-4caf-aad9-7b4ff0819a6c)

so by default I don't see a difference.
The fact that on  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd there is a difference as there they use a special css:
```
kbd {
  background-color: #eee;
  border-radius: 3px;
  border: 1px solid #b4b4b4;
  box-shadow:
    0 1px 1px rgba(0, 0, 0, 0.2),
    0 2px 0 0 rgba(255, 255, 255, 0.7) inset;
  color: #333;
  display: inline-block;
  font-size: 0.85em;
  font-weight: 700;
  line-height: 1;
  padding: 2px 4px;
  white-space: nowrap;
} 
```

Doxygen has currently no possibility to distinguish to specify something special for `<kbb>`-
This is not a bug but by design, but it might be a good enhancement (also taking the other output formats into consideration).

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #11211

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
